pkg/cluster/cluster.go

```diff
@@ -102,22 +102,25 @@ func new(config Config, s *spec.ClusterSpec, stopC <-chan struct{}, wg *sync.Wai
 		status:  &Status{},
 	}
 
-	if err := c.prepareBackupAndRestore(); err != nil {
-		if err == errBackupUnsetRestoreSet {
-			c.logger.Errorf("%v. This is not allowed. Cluster to be dead.", err)
+	if isNewCluster {
```

Comment:
The logging can be very annoying since the resync is expected. The controller will spam this logging.

One thing we should do is to voluntarily stop the controller and re-watch from the head revision to avoid the out of date event. After we have that mechinism, we can add logging here.